core/deviceadaptor.cpp

```diff
@@ -76,16 +76,15 @@ RingBufferBase* AdaptedSensorEntry::buffer() const
     return buffer_;
 }
 
-// TODO: correct initialization of screenBlanked is disabled since it
-// most probably has effect for testcase functionalities.
+// TODO: correct initialization of screenBlanked most probably has effect for testcase functionalities.
 DeviceAdaptor::DeviceAdaptor(const QString& id) :
     NodeBase(id),
     standbyOverride_(false),
-//#ifdef SENSORFW_MCE_WATCHER
-//    screenBlanked_(!SensorManager::instance().MCEWatcher()->displayEnabled())
-//#else
+#ifdef SENSORFW_MCE_WATCHER
+    screenBlanked_(!SensorManager::instance().MCEWatcher()->displayEnabled())
```

Comment:
That just returns a bool at that point. MCEWatcher was recently changed to non blocking dbus call.